### ISSUE TITLE
Ignore some properties

### ISSUE THREAD

**Issue by fredericoregateiro** [StackExchange/Dapper#1033]
Hello,
I have a issue with dapper, i'm trying to call a SP 
``` csharp
await connection.ExecuteAsync("[ApiInsertDocumentHeader]",
              documents, commandType: CommandType.StoredProcedure);
```
the documents is a array of Document class:
``` csharp
public class Document 
{
    public int Id { get; set; }
......
    [NotMapped]
    public AddressStruct Address { get; set; }
 }
```
The SP parameters contains all the Document properties, except the Address.

I'm getting this error:
The member Address of type AddressStruct cannot be used as a parameter value

How can i tell dapper to ignore the Address property?

I'm using dapper 1.50.5, dotnet core 2.1 and SQL Server

**Comment by mgravell** [comment]
Right now, the easiest way would be to send an arts value of

    new { obj.Id, obj.Name, obj.Foo, ... }

for all the required members. The problem is that dapper doesn't know the
schema of your SP, so it has to assume to send everything. I'm open to
better ideas, though. We could detect the [NotMapped], for example. Is that
part of dapper contribute/rainbow?

**Comment by fredericoregateiro** [comment]
Thanks,
If the dapper could detect the [NotMapped] would be great, and simple to use.
This code is from the EF we are moving to dapper, and the attribute it's from the System.ComponentModel.Annotations.dll

**Comment by NickCraver** [comment]
Tagging for #722 - I'm not sure ignored if something we've though about much, but I agree makes sense to support.

**Comment by amosvanhorn** [comment]
I agree - I'd love to see a  [DapperIgnore] property attribute to easily exclude a class property from all Dapper operations.
